tagsets/orsee_mysql.php

```diff
@@ -247,5 +247,121 @@ function dump_array($array,$title="",$dolang=true) {
     echo '</TABLE>';
 }
 
+function check_database_upgrade() {
+    global $settings, $system__database_version;
+
+    if (!isset($settings['database_version'])) {
+        $settings['database_version']='2017112700';
```

Comment:
You can safely use 0 here rather than actual date :)

Reply:
Ok.